pysat/tests/test_custom.py

```diff
@@ -43,11 +43,42 @@ def setup(self):
                                          clean_level='clean')
         self.testInst.load(2008, 1)
         self.ncols = len(self.testInst.data.columns)
+        self.out = None
 
     def teardown(self):
         """Runs after every method to clean up previous testing.
         """
-        del self.testInst, self.ncols
+        del self.testInst, self.ncols, self.out
+
+    def test_basic_repr(self):
+        """The repr output will match the str output"""
+        self.out = self.testInst.custom.__repr__()
+        assert isinstance(self.out, str)
+        assert self.out.find("functions applied") > 0
+
+    def test_basic_str(self):
+        """Check for lines from each decision point in str"""
+        self.out = self.testInst.custom.__str__()
+        assert isinstance(self.out, str)
+        # No custom functions
+        assert self.out.find('0 applied') > 0
+
+    def test_basic_str_w_function(self):
+        """Check for lines from each decision point in str"""
+        def mult_data(inst, mult, dkey="mlt"):
```

Comment:
The custom mult_data function isn't being run.

Reply:
Yup, it doesn't need to be run.  It just needs to be assigned.  This could be cleaned up by defining this function in setup or outside the Class since there's lots of duplication of defining small custom routines in this test suite.  However, that's a separate issue that should be created.